CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 4.4.0 - TBD
+
+## Enhancements
+
+- Adds in-built appium server control for local testing [#202](https://github.com/bugsnag/maze-runner/pull/202)
+
```

Comment:
I've got a changed lined up too, which I was looking at a 4.3.1 for, but looks like we might be able to roll out together.

Reply:
In fact I've just released 4.3.1 for the benefit of Android - just rebase onto that.